CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+## Pull Request Process
+
+The following outlines the general rules we follow:
+
+1. All PRs should have the appropriate testcases when possible. For example, bug fixes could include tests that demonstrates the issue w/o your fix.
```

Comment:
I guess this may change in close future.
The open questions:
- what kind of tests do we require, functional or unit? As I talked to @shaded-enmity, most probably functional test. And unit tests as a best practice, i.e. if they make sense.
- I mention _"testcases identified in the actor's documentation"_ but we haven't decided yet, how we will want actor developers to describe the testcases, if at all.

So for now, maybe this wording may be better:
_"All PRs dealing with actors must include functional tests covering the behaviour mentioned in the actor's description. Also, bug fixes should include tests that cover the fixed issue.""_